charts/newrelic-infrastructure-v3/README.md

```diff
@@ -44,7 +44,7 @@ Kubernetes: `>=1.16.0-0`
 | images.forwarder.repository | string | `"newrelic/k8s-events-forwarder"` | Image for the agent sidecar. |
 | images.forwarder.tag | string | `"1.20.5"` | Tag for the agent sidecar. |
 | images.integration.repository | string | `"newrelic/nri-kubernetes"` | Image for the kubernetes integration. |
-| images.integration.tag | string | `"0.1.2"` | Tag for the kubernetes integration. |
+| images.integration.tag | string | `Matching <Chart.appVersion>` | Tag for the kubernetes integration. |
```

Comment:
Is this true in the end? From what I can see, we have `default | .Chart.AppVersion` but that will never evaluate as we populate a tag by default.

Reply:
I was waiting for helm-docs to fix it